dockstore-webservice/src/main/java/io/dockstore/webservice/core/SourceControlConverter.java

```diff
@@ -39,6 +39,10 @@ public String convertToDatabaseColumn(SourceControl attribute) {
 
     @Override
     public SourceControl convertToEntityAttribute(String dbData) {
+        if (dbData == null || dbData.isEmpty()) {
+            LOG.info("Cannot convert null or empty source control");
+            return null;
+        }
```

Comment:
For line 47/51, was like this, but message is wrong; not a token type but a SourceControl